backend/src/agents/agent.py

```diff
@@ -45,15 +45,18 @@ def __get_action(self, utterance: str) -> Action_and_args:
 
         try:
             chosen_tool = extract_tool(response["tool_name"], self.tools)
+            logger.info(f"USER - Chosen tool: {chosen_tool.name}")

```

Comment:
Do we need this log and the one on line 50 if we also have the log on line 54?

Reply:
Removed this logger.